yarn.lock

```diff
@@ -8704,6 +8910,22 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@wordpress/element@npm:4.20.0, @wordpress/element@npm:^4.1.1, @wordpress/element@npm:^4.11.0, @wordpress/element@npm:^4.19.0, @wordpress/element@npm:^4.5.0, @wordpress/element@npm:^4.7.0":
```

Comment:
Same as for other `@wordpress/a11y` above. I'm concerned that some of those packages could have unexpected effects on the rest of Calypso. It's nice that our e2e tests pass, but then they don't provide a 100% coverage. WDYT?